### ISSUE TITLE
Cannot set property '0' of undefined

### ISSUE THREAD

**Issue by sergeyzwezdin** [wallabyjs/public#2466]
### Issue description or question

For an unknown reason, I started to get the following error from Wallaby:
```
Cannot set property '0' of undefined
```

I use `jest@26.0.1` with typescript. Running jest from command line doesn't fail with an error.

![image](https://user-images.githubusercontent.com/800755/86084739-0f7f3d80-bab7-11ea-97fc-dfeaed89b190.png)

### Wallaby diagnostics report

```
{
  editorVersion: '1.46.1',
  pluginVersion: '1.0.212',
  editorType: 'VSCode',
  osVersion: 'darwin 19.5.0',
  nodeVersion: 'v12.14.0',
  coreVersion: '1.0.920',
  config: {
    diagnostics: {
      jest: {
        config: {
          configs: [
            {
              automock: false,
              cache: true,
              cacheDirectory: '/private/var/folders/21/4bvtsw8s5zz3l_vrgpvwg6kw0000gn/T/jest_dx',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/Sources/Personal/test-backend',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: undefined,
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: {},
              haste: { computeSha1: false, throwOnModuleCollision: false },
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'js', 'json', 'jsx', 'ts', 'tsx', 'node' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'fb67eb5c6faeaede0c3fc5b61615b9da',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: undefined,
              restoreMocks: false,
              rootDir: '<homeDir>/Sources/Personal/test-backend',
              roots: [ '<homeDir>/Sources/Personal/test-backend' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/Sources/Personal/test-backend/node_modules/jest-environment-node/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/Sources/Personal/test-backend/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [ [ '^.+\\.tsx?$', '<homeDir>/Sources/Personal/test-backend/node_modules/ts-jest/dist/index.js', {} ] ],
              transformIgnorePatterns: [ '/node_modules/' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            collectCoverage: false,
            collectCoverageFrom: [],
            collectCoverageOnlyFrom: undefined,
            coverageDirectory: '<homeDir>/Sources/Personal/test-backend/coverage',
            coverageProvider: 'babel',
            coverageReporters: [ 'json', 'text', 'lcov', 'clover' ],
            coverageThreshold: undefined,
            detectLeaks: undefined,
            detectOpenHandles: undefined,
            enabledTestsMap: undefined,
            errorOnDeprecated: false,
            expand: false,
            filter: undefined,
            findRelatedTests: false,
            forceExit: false,
            globalSetup: undefined,
            globalTeardown: undefined,
            json: false,
            lastCommit: false,
            listTests: undefined,
            logHeapUsage: false,
            maxConcurrency: 5,
            maxWorkers: 15,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: undefined,
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: false,
            onlyFailures: false,
            outputFile: undefined,
            passWithNoTests: undefined,
            projects: [],
            replname: undefined,
            reporters: undefined,
            rootDir: '<homeDir>/Sources/Personal/test-backend',
            runTestsByPath: false,
            silent: undefined,
            skipFilter: false,
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: undefined,
            testSequencer: '<homeDir>/Sources/Personal/test-backend/node_modules/@jest/test-sequencer/build/index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: undefined,
            watch: false,
            watchAll: false,
            watchPlugins: undefined,
            watchman: true
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [],
            watchPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|/\\.cache/|/\\.idea/|/\\.vscode/|/\\.git/|/\\.gitlab/' ],
            testPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|/\\.cache/|/\\.idea/|/\\.vscode/|/\\.git/|/\\.gitlab/' ],
            testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)' ],
            testRegex: []
          }
        }
      }
    },
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [],
    globalSetup: false,
    micromatch: true,
    files: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|/\\.cache/|/\\.idea/|/\\.vscode/|/\\.git/|/\\.gitlab/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\/\.cache\/|\/\.idea\/|\/\.vscode\/|\/\.git\/|\/\.gitlab\//,
        ignore: true,
        trigger: true,
        load: true
      },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: true, trigger: true, load: true },
      { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: true, trigger: true, load: true }
    ],
    tests: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|/\\.cache/|/\\.idea/|/\\.vscode/|/\\.git/|/\\.gitlab/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\/\.cache\/|\/\.idea\/|\/\.vscode\/|\/\.git\/|\/\.gitlab\//,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 },
      { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 3 }
    ],
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    extensions: {},
    env: { type: 'node', params: {}, runner: '<homeDir>/.nvm/versions/node/v12.14.0/bin/node', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#1885170066'
  },
  packageJSON: {
    dependencies: { 'aws-sdk': '^2.700.0', 'body-parser': '^1.19.0', express: '^4.17.1', nanoid: '^3.1.10', 'yandex-cloud-functions-router': '^1.3.2' },
    devDependencies: {
      '@types/aws-sdk': '^2.7.0',
      '@types/express': '^4.17.6',
      '@types/jest': '^26.0.0',
      '@types/node': '^14.0.5',
      dotenv: '^8.2.0',
      jest: '^26.0.1',
      nodemon: '^2.0.4',
      prettier: '^2.0.5',
      'ts-jest': '^26.1.0',
      'ts-node': '^8.10.1',
      tslint: '^6.1.2',
      'tslint-config-airbnb': '^5.11.2',
      typescript: '^3.9.5'
    }
  },
  fs: { numberOfFiles: 108 },
  debug: [
    '2020-06-30T02:50:22.629Z angular/cli config Angular CLI not found.\n',
    '2020-06-30T02:50:22.679Z jest/config Detected Jest.\n',
    '2020-06-30T02:50:22.679Z jest/config Configured Jest.\n',
    '2020-06-30T02:50:22.680Z project Wallaby Node version: v12.14.0\n',
    '2020-06-30T02:50:22.680Z project Wallaby config: <homeDir>/Sources/Personal/test-backend/auto.detect\n',
    '2020-06-30T02:50:23.429Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.212/projects/445160af6eb598d6\n',
    '2020-06-30T02:50:23.438Z uiService Listening port 51235\n',
    '2020-06-30T02:50:23.450Z workers Parallelism for initial run: 14, for regular run: 7\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #0\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #1\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #2\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #3\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #4\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #5\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #6\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #7\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #8\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #9\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #10\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #11\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #12\n',
    '2020-06-30T02:50:23.450Z workers Starting run worker instance #13\n',
    '2020-06-30T02:50:23.450Z workers Web server is listening at 62890\n',
    '2020-06-30T02:50:23.451Z project File cache requires some updates, waiting required files from IDE\n',
    '2020-06-30T02:50:23.558Z project Stopping process pool\n',
    '2020-06-30T02:50:23.560Z project Test run started; run priority: 3\n',
    '2020-06-30T02:50:23.562Z project Running all tests\n',
    '2020-06-30T02:50:23.565Z workers Starting test run, priority: 3\n',
    '2020-06-30T02:50:23.565Z workers Distributing tests between 14 workers\n',
    '2020-06-30T02:50:23.566Z workers Running tests in parallel\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #0, session #byy84]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #1, session #1pcmc]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #2, session #tmnza]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #3, session #kbyw0]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #4, session #dbxvr]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #5, session #g0tub]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #6, session #me20k]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #7, session #lkv3u]\n',
    '2020-06-30T02:50:23.567Z nodeRunner Starting sandbox [worker #8, session #k8lb3]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #0, session #byy84]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #1, session #1pcmc]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #2, session #tmnza]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #3, session #kbyw0]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #4, session #dbxvr]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #5, session #g0tub]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #6, session #me20k]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #7, session #lkv3u]\n',
    '2020-06-30T02:50:23.568Z nodeRunner Preparing sandbox [worker #8, session #k8lb3]\n',
    '2020-06-30T02:50:23.621Z workers Started run worker instance (delayed) #0\n',
    '2020-06-30T02:50:23.622Z nodeRunner Prepared sandbox [worker #0, session #byy84]\n',
    '2020-06-30T02:50:23.622Z workers [worker #0, session #byy84] Running tests in sandbox\n',
    '2020-06-30T02:50:23.625Z workers Started run worker instance (delayed) #1\n',
    '2020-06-30T02:50:23.626Z nodeRunner Prepared sandbox [worker #1, session #1pcmc]\n',
    '2020-06-30T02:50:23.626Z workers [worker #1, session #1pcmc] Running tests in sandbox\n',
    '2020-06-30T02:50:23.628Z workers Started run worker instance (delayed) #2\n',
    '2020-06-30T02:50:23.628Z nodeRunner Prepared sandbox [worker #2, session #tmnza]\n',
    '2020-06-30T02:50:23.628Z workers [worker #2, session #tmnza] Running tests in sandbox\n',
    '2020-06-30T02:50:23.632Z workers Started run worker instance (delayed) #4\n',
    '2020-06-30T02:50:23.632Z nodeRunner Prepared sandbox [worker #4, session #dbxvr]\n',
    '2020-06-30T02:50:23.632Z workers [worker #4, session #dbxvr] Running tests in sandbox\n',
    '2020-06-30T02:50:23.636Z workers Started run worker instance (delayed) #3\n',
    '2020-06-30T02:50:23.637Z nodeRunner Prepared sandbox [worker #3, session #kbyw0]\n',
    '2020-06-30T02:50:23.637Z workers [worker #3, session #kbyw0] Running tests in sandbox\n',
    '2020-06-30T02:50:23.637Z workers Started run worker instance (delayed) #5\n',
    '2020-06-30T02:50:23.638Z nodeRunner Prepared sandbox [worker #5, session #g0tub]\n',
    '2020-06-30T02:50:23.638Z workers [worker #5, session #g0tub] Running tests in sandbox\n',
    '2020-06-30T02:50:23.642Z workers Started run worker instance (delayed) #8\n',
    '2020-06-30T02:50:23.642Z nodeRunner Prepared sandbox [worker #8, session #k8lb3]\n',
    '2020-06-30T02:50:23.642Z workers [worker #8, session #k8lb3] Running tests in sandbox\n',
    '2020-06-30T02:50:23.643Z workers Started run worker instance (delayed) #6\n',
    '2020-06-30T02:50:23.643Z nodeRunner Prepared sandbox [worker #6, session #me20k]\n',
    '2020-06-30T02:50:23.643Z workers [worker #6, session #me20k] Running tests in sandbox\n',
    '2020-06-30T02:50:23.646Z workers Started run worker instance (delayed) #7\n',
    '2020-06-30T02:50:23.646Z nodeRunner Prepared sandbox [worker #7, session #lkv3u]\n',
    '2020-06-30T02:50:23.646Z workers [worker #7, session #lkv3u] Running tests in sandbox\n',
    '2020-06-30T02:50:23.647Z workers Started run worker instance (delayed) #9\n',
    '2020-06-30T02:50:23.651Z workers Started run worker instance (delayed) #11\n',
    '2020-06-30T02:50:23.653Z workers Started run worker instance (delayed) #10\n',
    '2020-06-30T02:50:23.655Z workers Started run worker instance (delayed) #12\n',
    '2020-06-30T02:50:23.656Z workers Started run worker instance (delayed) #13\n',
    '2020-06-30T02:50:37.853Z workers [k8lb3] Loaded unknown number of test(s)\n',
    '2020-06-30T02:50:37.854Z workers [k8lb3] Test executed: TODO\n',
    '2020-06-30T02:50:37.873Z workers [k8lb3] Run 1 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:37.874Z workers [k8lb3] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:39.746Z workers [me20k] Loaded unknown number of test(s)\n',
    '2020-06-30T02:50:39.747Z workers [me20k] Test executed: searches the data\n',
    '2020-06-30T02:50:39.747Z workers [me20k] Test executed: searches the data by page\n',
    '2020-06-30T02:50:39.747Z workers [me20k] Test executed: returns empty result\n',
    '2020-06-30T02:50:39.759Z workers [me20k] Run 3 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:39.761Z workers [me20k] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:39.968Z workers [lkv3u] Loaded unknown number of test(s)\n',
    '2020-06-30T02:50:39.969Z workers [lkv3u] Test executed: searches the data\n',
    '2020-06-30T02:50:39.969Z workers [lkv3u] Test executed: searches the data (5 items)\n',
    '2020-06-30T02:50:39.970Z workers [lkv3u] Test executed: returns empty result\n',
    '2020-06-30T02:50:39.984Z workers [lkv3u] Run 3 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:39.986Z workers [lkv3u] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:40.373Z workers [g0tub] Loaded unknown number of test(s)\n',
    '2020-06-30T02:50:40.373Z workers [g0tub] Test executed: sends email\n',
    '2020-06-30T02:50:40.374Z workers [g0tub] Test executed: sends email without data and bcc\n',
    '2020-06-30T02:50:40.386Z workers [g0tub] Run 2 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:40.387Z workers [g0tub] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:40.439Z workers [dbxvr] Loaded unknown number of test(s)\n',
    '2020-06-30T02:50:40.440Z workers [dbxvr] Test executed: unsubscribes email (specific post)\n',
    '2020-06-30T02:50:40.449Z workers [dbxvr] Run 1 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:40.450Z workers [dbxvr] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:40.535Z workers [kbyw0] Loaded unknown number of test(s)\n',
    '2020-06-30T02:50:40.535Z workers [kbyw0] Test executed: posts comment\n',
    '2020-06-30T02:50:40.536Z workers [kbyw0] Test executed: sends email when error throws\n',
    '2020-06-30T02:50:40.536Z workers [kbyw0] Test executed: sends email when unable to post comment\n',
    "2020-06-30T02:50:40.548Z workers Sandbox (active) [tmnza] error: Cannot set property '0' of undefined\n",
    '2020-06-30T02:50:40.560Z workers [kbyw0] Run 3 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:40.561Z workers [kbyw0] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:40.563Z workers [tmnza] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:40.564Z workers [tmnza] Sandbox is responsive, closing it\n',
    "2020-06-30T02:50:40.891Z workers Sandbox (active) [byy84] error: Cannot set property '0' of undefined\n",
    '2020-06-30T02:50:40.900Z workers [byy84] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:40.901Z workers [byy84] Sandbox is responsive, closing it\n',
    "2020-06-30T02:50:40.918Z workers Sandbox (active) [1pcmc] error: Cannot set property '0' of undefined\n",
    '2020-06-30T02:50:40.926Z workers [1pcmc] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:50:40.927Z workers [1pcmc] Sandbox is responsive, closing it\n',
    '2020-06-30T02:50:40.928Z workers Merging parallel test run results\n',
    '2020-06-30T02:50:40.937Z project Test run finished\n',
    '2020-06-30T02:50:40.937Z project Processed console.log entries\n',
    '2020-06-30T02:50:40.937Z project Processed loading sequences\n',
    '2020-06-30T02:50:40.938Z project Processed executed tests\n',
    '2020-06-30T02:50:40.948Z project Processed code coverage\n',
    '2020-06-30T02:50:40.990Z project Test run result processed and sent to IDE\n',
    '2020-06-30T02:51:05.102Z fs File changed in editor: email-app/handlers/http/httpMailgunUnsubscribeComments.ts\n',
    '2020-06-30T02:51:05.103Z extended-core New document or complex document change\n',
    '2020-06-30T02:51:05.108Z project Test run started; run priority: 2\n',
    '2020-06-30T02:51:05.108Z testTask Test files from affected: 2, from deleted or manually requested: 0, from recently changed: 0, from loaded by: 1, from failing: 0\n',
    '2020-06-30T02:51:05.110Z workers Starting test run, priority: 2\n',
    '2020-06-30T02:51:05.110Z workers Distributing tests between 7 workers\n',
    '2020-06-30T02:51:05.110Z workers Running tests in parallel\n',
    '2020-06-30T02:51:05.110Z nodeRunner Starting sandbox [worker #0, session #86jwv]\n',
    '2020-06-30T02:51:05.110Z nodeRunner Starting sandbox [worker #1, session #8f426]\n',
    '2020-06-30T02:51:05.110Z nodeRunner Starting sandbox [worker #2, session #xfml5]\n',
    '2020-06-30T02:51:05.110Z nodeRunner Starting sandbox [worker #3, session #vdsdu]\n',
    '2020-06-30T02:51:05.110Z nodeRunner Starting sandbox [worker #4, session #kfk5i]\n',
    '2020-06-30T02:51:05.110Z nodeRunner Starting sandbox [worker #5, session #7vbxu]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Preparing sandbox [worker #0, session #86jwv]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Preparing sandbox [worker #1, session #8f426]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Preparing sandbox [worker #2, session #xfml5]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Preparing sandbox [worker #3, session #vdsdu]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Preparing sandbox [worker #4, session #kfk5i]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Preparing sandbox [worker #5, session #7vbxu]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Prepared sandbox [worker #0, session #86jwv]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Prepared sandbox [worker #1, session #8f426]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Prepared sandbox [worker #2, session #xfml5]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Prepared sandbox [worker #3, session #vdsdu]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Prepared sandbox [worker #4, session #kfk5i]\n',
    '2020-06-30T02:51:05.111Z nodeRunner Prepared sandbox [worker #5, session #7vbxu]\n',
    '2020-06-30T02:51:05.111Z workers [worker #0, session #86jwv] Running tests in sandbox\n',
    '2020-06-30T02:51:05.112Z workers [worker #1, session #8f426] Running tests in sandbox\n',
    '2020-06-30T02:51:05.112Z workers [worker #2, session #xfml5] Running tests in sandbox\n',
    '2020-06-30T02:51:05.112Z workers [worker #3, session #vdsdu] Running tests in sandbox\n',
    '2020-06-30T02:51:05.113Z workers [worker #4, session #kfk5i] Running tests in sandbox\n',
    '2020-06-30T02:51:05.113Z workers [worker #5, session #7vbxu] Running tests in sandbox\n',
    "2020-06-30T02:51:05.292Z workers Sandbox (active) [7vbxu] error: Cannot set property '0' of undefined\n",
    '2020-06-30T02:51:05.306Z workers [7vbxu] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:51:05.308Z workers [7vbxu] Sandbox is responsive, closing it\n',
    "2020-06-30T02:51:05.407Z workers Sandbox (active) [xfml5] error: Cannot set property '0' of undefined\n",
    "2020-06-30T02:51:05.419Z workers Sandbox (active) [8f426] error: Cannot set property '0' of undefined\n",
    "2020-06-30T02:51:05.432Z workers Sandbox (active) [kfk5i] error: Cannot set property '0' of undefined\n",
    "2020-06-30T02:51:05.438Z workers Sandbox (active) [vdsdu] error: Cannot set property '0' of undefined\n",
    "2020-06-30T02:51:05.441Z workers Sandbox (active) [86jwv] error: Cannot set property '0' of undefined\n",
    '2020-06-30T02:51:05.449Z workers [xfml5] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:51:05.451Z workers [xfml5] Sandbox is responsive, closing it\n',
    '2020-06-30T02:51:05.467Z workers [8f426] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:51:05.469Z workers [8f426] Sandbox is responsive, closing it\n',
    '2020-06-30T02:51:05.471Z workers [kfk5i] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:51:05.473Z workers [vdsdu] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:51:05.473Z workers [86jwv] Run 0 test(s), skipped 0 test(s)\n',
    '2020-06-30T02:51:05.473Z workers [kfk5i] Sandbox is responsive, closing it\n',
    '2020-06-30T02:51:05.476Z workers [vdsdu] Sandbox is responsive, closing it\n',
    '2020-06-30T02:51:05.477Z workers [86jwv] Sandbox is responsive, closing it\n',
    '2020-06-30T02:51:05.477Z workers Merging parallel test run results\n',
    '2020-06-30T02:51:05.486Z project Test run finished\n',
    '2020-06-30T02:51:05.486Z project Processed console.log entries\n',
    '2020-06-30T02:51:05.486Z project Processed loading sequences\n',
    '2020-06-30T02:51:05.486Z project Processed executed tests\n',
    '2020-06-30T02:51:05.490Z project Processed code coverage\n',
    '2020-06-30T02:51:05.496Z project Test run result processed and sent to IDE\n',
    '2020-06-30T02:51:05.552Z fs File changed: email-app/handlers/http/httpMailgunUnsubscribeComments.ts\n',
    '2020-06-30T02:51:05.553Z fs File changed in editor: email-app/handlers/http/httpMailgunUnsubscribeComments.ts\n',
    '2020-06-30T02:51:05.553Z fs No changes detected for email-app/handlers/http/httpMailgunUnsubscribeComments.ts\n',
    '2020-06-30T02:51:05.553Z extended-core File was not changed, but file markers may need to be synced\n'
  ]
}
```

**Comment by ArtemGovorov** [comment]
Thanks, can you also please:

- stop Wallaby,
- delete `<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.212/projects/445160af6eb598d6` folder,
- delete `/private/var/folders/21/4bvtsw8s5zz3l_vrgpvwg6kw0000gn/T/jest_dx` folder,
- start Wallaby,
- reproduce the error by editing the `email-app/handlers/http/httpMailgunUnsubscribeComments.ts` file, 
- run Copy Diagnostics Report command and share the report?

**Comment by sergeyzwezdin** [comment]
That is strange, now I cannot reproduce it. I will observe it and will let you know.
Thank you for your assistance!

**Comment by ArtemGovorov** [comment]
Awesome, thanks for the update. It's likely then that clearing Jest's cache (`delete /private/var/folders/21/4bvtsw8s5zz3l_vrgpvwg6kw0000gn/T/jest_dx folder` had helped). However, running `jest --clearCache` in the project's root should have helped too then (as it is essentially the same). 

Will close the issue for now, if the issue shows up again, please let us know.

**Comment by davidveen** [comment]
I'm having a similar issue, where Wallaby reports 'Cannot set property '0' of undefined' on a line importing from @ngrx/store (v10)
![afbeelding](https://user-images.githubusercontent.com/16062746/100344245-d0f79b80-2fe0-11eb-80f8-edd70b240c17.png)

Interestingly, clearing the jest cache as suggested temporarily solves the problem. Then after making _any_ change to the test suite, the error immediately pops up again.

Jest reports no errors.

Any thoughts? Should I create a ticket with debug info?

**Comment by ArtemGovorov** [comment]
@davidveen Yes, please create a new ticket with the diagnostics report, we'll try to set up a sample project to reproduce the issue (unless you already have a sample project where we can reproduce it, in which case it will significantly speed up the investigation). One thing that is worth trying before you create the ticket is to try both: stop Wallaby and then run `jest --clearCache` plus delete the `<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-VERSION/projects` folder.

**Comment by davidveen** [comment]
Removing the projects folder did the trick! Thanks a ton @ArtemGovorov
